gatsby-node.js

```diff
@@ -112,29 +116,45 @@ exports.createPages = ({ graphql, actions }) => {
         });
 
         docPages.forEach(page => {
-          createPage({
-            path: `/learn/${page.slug}`,
-            component: docTemplate,
-            context: {
-              slug: page.slug,
-              next: page.next,
-              previous: page.previous,
-              relativePath: page.relativePath,
-              navigationData,
-            },
-          });
-          if (page.slug === 'introduction-to-nodejs') {
-            createPage({
-              path: `/learn`,
-              component: docTemplate,
-              context: {
-                slug: page.slug,
-                next: page.next,
-                previous: page.previous,
-                relativePath: page.relativePath,
-                navigationData,
-              },
-            });
+          switch (page.slug) {
+            case 'installing-via-package-manager':
```

Comment:
I don't hate this at all, but we may want to have a dedicated spot in the docs for these files to live instead of just one magical, specially named markdown file.

Reply:
Ah I see, you did! May be good to document once we get this done.

Reply:
Would actually love to see this with a dedicated file for each platform – keeps things better organized and makes adding more relatively trivial.

Reply:
Wow… I get it, it was an evolution for me as well :)

I'll be honest, it was the first time to hear of Gatsby, so I was happy to just open the space up with allotted for this project until we figure out if/how to capitalize moving forward.

The Why's on my mind for all matters of content all living in `md`'s:

- Better flows for i18n and a11y — ie if it pans out we can explore how other parts of the project can similarly benefit.

- Better flows for editorial ops — ie PRs working on `md` content, that's like editing a word document in word :)

- Better flows for contribution and syndication — ie "edit on…" and feeds

- Single-source-of-truth, separation of concerns… all those sometimes-watery to-be inlined considerations.

Next steps: I'd like us to work out the details and docs once we close this round of deliverables.

Reply:
> dedicated file for each platform

I think we need to discuss pros/cons here — ie but not literally here. This has many "winged" things and takes a good discussion to make sure we give due considerations moving forward.

I can set us up on zoom, and relatively flexible time-wise… just let me know what works.

Reply:
@amiller-gh I really hesitate to move forward before addressing this further in a zoom call please.